src/components/CashoutModal.tsx

```diff
@@ -67,32 +71,33 @@ export default function DepositModal(): ReactElement {
       <Snackbar anchorOrigin={{ vertical: 'top', horizontal: 'center' }} open={showToast} message={toastContent} />
       <Dialog open={open} onClose={handleClose} aria-labelledby="form-dialog-title">
         <DialogTitle id="form-dialog-title">Cashout Cheque</DialogTitle>
-        {loadingCashout ? (
-          <Container style={{ textAlign: 'center', padding: '50px' }}>
-            <CircularProgress />
-          </Container>
-        ) : (
-          <DialogContent>
-            <DialogContentText style={{ marginTop: '20px' }}>
-              Specify the peer Id of the peer you would like to cashout.
-            </DialogContentText>
-            <Input
-              autoFocus
-              margin="dense"
-              id="peerId"
-              type="text"
-              placeholder="Peer Id"
-              fullWidth
-              onChange={e => setPeerId(e.target.value)}
-            />
-          </DialogContent>
-        )}
+        <DialogContent>
+          <DialogContentText style={{ marginTop: '20px', overflowWrap: 'break-word' }}>
+            {loadingCashout && (
+              <>
+                <span>
+                  Cashing out <strong>{fromBZZbaseUnit(uncashedAmount).toFixed(7)}</strong> from Peer{' '}
```

Comment:
It would be nice to use constants instead of number literals, especially if they are repeated.

Reply:
Apologies, what do you mean here? Extract the `fromZZZBaseUnit(...)` into the hooks directly and just display it here? So it would be:

```suggestion
                  Cashing out <strong>{uncashedAmount}</strong> from Peer{' '}
```

Reply:
I mean instead of using `.toFixed(7)` it would be better to use `.toFixed(BZZ_DIGITS)` (or whatever name makes sense)